proxy-node-gateway/src/main/java/uk/gov/ida/notification/resources/EidasAuthnRequestResource.java

```diff
@@ -52,7 +53,7 @@ public EidasAuthnRequestResource(
     @GET
     @Path(Urls.GatewayUrls.GATEWAY_EIDAS_AUTHN_REQUEST_REDIRECT_PATH)
     public View handleRedirectBinding(
-            @QueryParam(SamlFormMessageType.SAML_REQUEST) String encodedEidasAuthnRequest,
+            @QueryParam(SamlFormMessageType.SAML_REQUEST) @ValidBase64SamlAuthnRequest String encodedEidasAuthnRequest,
```

Comment:
The  `@ValidBase64SamlAuthnRequest` validator used `Base64SamlAuthnRequestValidator` for validation, which in turn decodes and parses the request - this is happening in gateway and we want to remove all saml parsing from this microservice. Think we should discuss this as a team.

Reply:
Removed and replaced with a simple Base64 check.